pr_agent/servers/github_app.py

```diff
@@ -124,19 +126,20 @@ async def handle_request(body: Dict[str, Any], event: str):
                 if pull_request.get("created_at") == pull_request.get("updated_at"):
                     # avoid double reviews when opening a PR for the first time
                     return {}
-            logging.info(f"Performing review because of event={event} and action={action}")
+            get_logger().info(f"Performing review because of event={event} and action={action}")
```

Comment:
It would be beneficial to add some context to the log messages. This can be done by including additional information in the log messages, such as the PR or issue number, the user who triggered the event, etc. This will make it easier to trace back the logs to the specific events or actions. [medium]